.gitignore

```diff
@@ -0,0 +1,3 @@
+*.jpg
+*.ico
+*.png
```

Comment:
Put all images in a single folder like `image` and add the `image/` into the .gitignore file.

Reply:
ok